src/components/member-profile/contribution-type/index.js

```diff
@@ -115,23 +85,16 @@ const ContributionType = (props) => {
       <div className={showMoreContentClass}>
         {type !== 'Active tasks' ? (
           <div>
-            {devUser
-              ? renderContributions(
-                  contributions,
-                  fullName,
-                  imageLink,
-                  devUser,
-                  type
-                )
-              : renderContributions(
-                  contributions,
-                  fullName,
-                  imageLink,
-                  devUser
-                )}
+            {renderContributions(
+              contributions,
+              fullName,
+              imageLink,
+              devUser,
+              type
+            )}
```

Comment:
Can you please verify here...As I am seeing previous changes
when devUser true then prop- type used

```
{devUser
              ? renderContributions(
                  contributions,
                  fullName,
                  imageLink,
                  devUser,
                  type
                )
              : renderContributions(
                  contributions,
                  fullName,
                  imageLink,
                  devUser
                )}
                
                ```

Reply:
No proptype is used in both the condition. PFA below screenshot.

![image](https://github.com/Real-Dev-Squad/website-members/assets/22474104/cea13fb7-8065-41a2-9571-0c76965f86f7)